BackgroundWorker/Program.cs

```diff
@@ -31,6 +31,15 @@
 // Learn more about configuring Swagger/OpenAPI at https://aka.ms/aspnetcore/swashbuckle
 builder.Services.AddEndpointsApiExplorer();
 builder.Services.AddSwaggerGen();
+builder.Services.AddCors(option =>
```

Comment:
can you explain what part requests it?

Reply:
pay attention that nothing like this is configured in [this](https://learn.microsoft.com/en-us/aspnet/core/tutorials/signalr?view=aspnetcore-8.0&WT.mc_id=dotnet-35129-website&tabs=visual-studio#configure-signalr) example

Reply:
> can you explain what part requests it?

While negotiation with background API UI was throwing cros policy error and was not able to intercommunicate with API hence we need to add cros policy